atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -250,8 +303,21 @@ def resolve_signing_intent(self):
         self.compose_config.set_signing_intent(signing_intent['name'])
 
     def forward_composes(self):
-        yum_repourls = [compose_info['result_repofile'] for compose_info in self.composes_info]
-        override_build_kwarg(self.workflow, 'yum_repourls', yum_repourls)
+        repos_by_arch = {}
+        # set overrides by arch if arches are available
+        for compose_info in self.composes_info:
+            for arch in compose_info.get('arches', []):
```

Comment:
What if "arches" is empty? Do we need to set `None` key in `repos_by_arch` ?

Reply:
The way I thought this was going to work:
if there are arches, then override is set specifically per arch
if there are not arches, then override is set generally (ie, None) for all repos in the compose.

Reply:
Fixed I think.